hw5.py

```diff
@@ -0,0 +1,8 @@
+#2
+print("Guess a number between 1 and 9")
+a = int(input("Enter a guess:"))
+while (a > 9) or (a < 1):
+    a = int(input("Enter a guess:"))
+else:
+    print("Well guessed!")
+
```

Comment:
requirements was not properly understood.
1. choose a random number from values between 1 and 9 (use random)
2. User will enter values until the value introduced is the same with the one from 1.